api-specs/Gateway-EE/3.3/kong-ee-3.3.yaml

```diff
@@ -1093,7 +1121,7 @@ components:
                   validator:
                     required: true
                     type: string
-              type: array
+              type: object
```

Comment:
`portal_developer_meta_fields` should be a string that presents a JSON object. Let's replace this field with:

```yaml
portal_developer_meta_fields:
  default: >-
    [{"label":"Full Name","title":"full_name","validator":{"required":true,"type":"string"}}]
  type: string
```

Reply:
Also, in the lines below:

```diff
@@ -1172,12 +1172,7 @@ components:
             portal_auto_approve: true
             portal_cors_origins:
               - string
-            portal_developer_meta_fields:
-              - label: Full Name
-                title: full_name
-                validator:
-                  required: true
-                  type: string
+            portal_developer_meta_fields: string
             portal_emails_from: string
             portal_emails_reply_to: string
             portal_invite_email: true
```